dask/dataframe/io/parquet.py

```diff
@@ -165,13 +164,19 @@ def to_parquet(path, df, encoding=default_encoding, compression=None,
         Destination directory for data.  Prepend with protocol like ``s3://``
         or ``hdfs://`` for remote data.
     df : Dask.dataframe
-    encoding : parquet_thrift.Encoding
+    encoding : parquet_thrift.Encoding (future use)
```

Comment:
I recommend that we remove it until its useful.  It's far easier to add options than to remove them.